kolibri/plugins/style_guide/assets/src/views/content/breadcrumbs/index.vue

```diff
@@ -1,29 +1,30 @@
 <template>
 
-  <page-template title="Breadcrumbs" :completed="false">
+  <page-template :completed="true">
 
-    <h2><code>{{ kBreadcrumbsApi.name }}</code> API</h2>
-    <component-docs :api="kBreadcrumbsApi" />
+    <h1>Channel breadcrumbs</h1>
```

Comment:
I kind of like topic tree more than channel.

Reply:
doesn't it go a level up from topics? e.g. first level is a list of channels, not topics